intents.rst

```diff
@@ -0,0 +1,136 @@
+Collects's Intent
+==================
+
+About Intent
```

Comment:
I think this section should focus on why someone might want to launch Collect from their app. Linking out to the Android docs is enough general background on Intents.